app-desktop/build.gradle.kts

```diff
@@ -16,9 +16,13 @@ kotlin {
             dependencies {
                 implementation(compose.desktop.currentOs)
                 implementation(projects.ui.root)
+                implementation(projects.di)
                 implementation(projects.ui.common)
                 implementation(libs.com.arkivanov.decompose)
                 implementation(libs.com.arkivanov.decompose.extensions.compose.jetbrains)
+                implementation(libs.io.insert.koin.core)
+                implementation(libs.kotlinx.coroutines.javafx)
```

Comment:
where do we need javafx for? isn't compose handling all ui already?

Reply:
In order to work with `Main` CoroutineDispatcher, I pulled in the JavaFX app dispatcher so I can pass in Main dispatcher through my decompose components. (The common Dispatchers.Main we use in android apps is from `kotlinx-coroutines-android`.

Alternatively, I can create my own custom "main", "default" and "io" dispatchers with some additional pool setup. I dont feel strongly.

Reply:
I guess we could think of having Dispatchers injected through DI, declared in a separate `common` module, with `expect/actual` declarations (for Main/IO/Default).

Reply:
+1 to injecting, I think you can inject coroutine context then construct a dispatcher from it

Reply:
@krizzu yes, that's along the lines of what I was thinking. I will close this loop in a follow-up PR.